sentry-actix/Cargo.toml

```diff
@@ -24,6 +24,7 @@ with_sentry_default = [
 [dependencies]
 sentry = { version = "0.20.1", path = "../sentry", default-features = false }
```

Comment:
I think we can change this to depend on `sentry-core` instead. (with a dev-dependency on `sentry` for doctests)